oss-cloud-frontend/src/components/ContributorsTable.vue

```diff
@@ -31,18 +31,33 @@
           item-key="username"
           show-expand
           class="elevation-2"
+          @click:row="expandRow"
```

Comment:
Is it required to have the whole row clickable resulting the expansion with contributions?

Reply:
Yes, @mjovanovic requested that feature.